### PR TITLE
Add X-Ray to whats new

### PR DESCRIPTION
**Pull request by zoltanbedi**

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 1]
```diff
@@ -157,3 +158,7 @@ Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELO
 In collaboration with Microsoft, we have improved the usability of our ADX datasource plugin by adding a visual query builder. The goal is to make it easier for users, regardless of their previous knowledge of writing KQL (Kusto Query Language) queries, to query and visualize their data.
 
 {{< docs-imagebox img="/img/docs/v72/adx-ds.png" max-width="800px" caption="ADX visual query builder" >}}
+
+### X-Ray data source plugin
+
+We are glad to announce our very first version of our data source plugin for AWS X-Ray. You can use the plugin to visualize traces, look at analytics tables, and insight summaries. For more information look at [plugin's page](https://grafana.com/grafana/plugins/grafana-x-ray-datasource).
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:164]:
```suggestion
We are pleased to announce our very first version of our data source plugin for AWS X-Ray. You can use this plugin to visualize traces, look at analytics tables, and see insight summaries. For more information, refer to the [X-Ray data source](https://grafana.com/grafana/plugins/grafana-x-ray-datasource) plugin page.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Some minor edits.

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ The main highlights are:
   - [**Report time range**]({{< relref "#report-time-range" >}})
   - [**Organization-wide report settings**]({{< relref "#organization-wide-report-settings" >}})
   - [**Report grid layout**]({{< relref "#report-grid-layout" >}})
+- [**X-Ray data source plugin**]({{< relref "#x-ray-data-source-plugin" >}})
```
Comment (achatterjee-grafana) [inline docs/sources/guides/whats-new-in-v7-2.md:35]: Needs proper alignment.

Comment (zoltanbedi) [reply docs/sources/guides/whats-new-in-v7-2.md:35]: I think this is aligned properly as it is not an enterprise feature.

**Review by oddlittlebird — APPROVED** [review]
LGTM!

**Comment by zoltanbedi** [comment]
> LGTM!

Thank you for your fast review.

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 3]
```diff
@@ -157,3 +158,7 @@ Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELO
 In collaboration with Microsoft, we have improved the usability of our ADX datasource plugin by adding a visual query builder. The goal is to make it easier for users, regardless of their previous knowledge of writing KQL (Kusto Query Language) queries, to query and visualize their data.
 
 {{< docs-imagebox img="/img/docs/v72/adx-ds.png" max-width="800px" caption="ADX visual query builder" >}}
+
+### X-Ray data source plugin
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:162]: I see the problem. @zoltanbedi, please update the TOC at the top to include the missing items, including the two headings above your entry.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/guides/whats-new-in-v7-2.md (hunk 3 above)
Comment (zoltanbedi) [reply docs/sources/guides/whats-new-in-v7-2.md:164]: Added the missing titles.

**Review by oddlittlebird — APPROVED** [review]
LGTM!

**Pull request merged**